components/x-podcast-launchers/src/generate-rss-url.js

```diff
@@ -0,0 +1,8 @@
+import nanoid from 'nanoid';
+
+const ACAST_RSS_URL = 'https://access.acast.cloud/rss';
```

Comment:
Given there is a prod URL too, maybe we should pass this in as a prop?

Reply:
Yeah could live in Vault maybe

Reply:
This is now a prop, so its up the apps how they pass this in